File: backend/src/main/scala/domains/auth/model/response/AuthAccountListItem.scala
package domains.auth.model.response

import domains.auth.model.EmailAddress
import domains.user.model.{DisplayName, Username}

final case class AuthAccountListItem(
  username: Username,
  displayName: DisplayName,
  email: EmailAddress,
  siteManager: Boolean,
  problemManager: Boolean
)

File: frontend/src/features/auth/model/response/AuthAccountListItem.ts
import type { EmailAddress } from '@/features/auth/model/EmailAddress'
import type { DisplayName } from '@/features/user/model/DisplayName'
import type { Username } from '@/features/user/model/Username'

export type AuthAccountListItem = {
  username: Username
  displayName: DisplayName
  email: EmailAddress
  siteManager: boolean
  problemManager: boolean
}
